sklearn/tests/test_multioutput.py

```diff
@@ -470,22 +470,23 @@ def test_classifier_chain_vs_independent_models():
     # Verify that an ensemble of classifier chains (each of length
     # N) can achieve a higher Jaccard similarity score than N independent
     # models
-    yeast = fetch_mldata('yeast')
-    X = yeast['data']
-    Y = yeast['target'].transpose().toarray()
-    X_train = X[:2000, :]
-    X_test = X[2000:, :]
-    Y_train = Y[:2000, :]
-    Y_test = Y[2000:, :]
+    X, y = make_classification(n_samples=1000,
```

Comment:
@jnothman Thanks. I think we need a random state here to ensure a certain result(In the previous test, the author's purpose is to ensure that different calculation methods get same result so random state is not needed). There are indeed some cases when ClassifierChain get wrose result.